### DISCUSSION TITLE
QUESTION: How to report the progress in the logs of docker containers?

### DISCUSSION THREAD

**Discussion by JoseMMerida** [futureverse/progressr#159] [Q&A]
I am not able to see the dumps of the progress in the logs, even when redirecting the output to stderr and stdout. There is a way to use this API to do that, or at least to gather the metrics to upload a front-end or so.

**Comment by HenrikBengtsson** [comment]
It's not 100% clear what your setup is, but I make a best guess.  You're probably running R in batch mode (= non-interactive mode). In that case, you need to set R option `progressr.enable` to TRUE (defaults to `interactive()`).  You can also set this option via environment variable `R_PROGRESSR_ENABLE`, cf. Section 'Environment variables that set R options' in <https://progressr.futureverse.org/reference/progressr.options.html>.

**Comment by JoseMMerida** [comment]
Thanks! I will take a look! :)

**Comment by JoseMMerida** [comment]
Actuallly I am using a function that handles the progress in a future function, due to its heavy cost. I al willing to track how it is doing.
